recipe/build.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/bash
 
-cmake -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
+mkdir -p build
+pushd build
+cmake ${CMAKE_ARGS} ..
```

Comment:
Probably a good idea to keep those
```suggestion

cmake ${CMAKE_ARGS} \
    -DCMAKE_INSTALL_PREFIX=$PREFIX \
    -DCMAKE_INSTALL_LIBDIR=lib \
    ..

```

Reply:
they are defined in the CTNG activation feedstock it seems.

Reply:
https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/9661e4c851131527fab6a895ceb567b5172b9d48/recipe/activate-gcc.sh#L150